py/selenium/webdriver/firefox/service.py

```diff
@@ -46,7 +47,14 @@ def __init__(self, executable_path: str = DEFAULT_EXECUTABLE_PATH,
             in the services' environment.
 
         """
-        log_file = open(log_path, "a+") if log_path else None
+        log_file = None
```

Comment:
Seems good to me; we could also allow `log_path` to become an (optional) length-two tuple that accepts something like `log_path: Union[str, Tuple[str, str]]` where the later case would be the `path, mode` passed to `open(...)`.  I'm ok with the current changes tho; seems to keep the flow but support an edge case.  Could we add a test for this?

Reply:
I like this solution a lot more so I changed it to that. I hope the change to the doc string is understandable.
I'm currently unable to run the tests locally but I'll try to write one anyway.